pdf_id/pdfid/pdfid.py

```diff
@@ -94,7 +94,7 @@ def C2BIP3(string):
 class cBinaryFile:
     def __init__(self, file, data=None):
         self.file = file
-        if data is not None:
+        if data is not None and not isinstance(data, list):
```

Comment:
Isn't the problem on line 803?
`xmlDoc = PDFiD(filename, options.all, options.extra, options.disarm, options.force, additional_keywords)`
when the definition is
`PDFiD(file, allNames=False, extraData=False, disarm=False, force=False, data=None, additional_keywords=[]):`
You are receiving the additional_keywords as "data". This fix may "work" but is not the clearest way to fix it.

Reply:
Good catch!